engine/commands/cortex_upd_cmd.cc

```diff
@@ -12,21 +13,38 @@
 
 namespace commands {
 
-namespace {
-const std::string kCortexBinary = "cortex-cpp";
-}
-
 CortexUpdCmd::CortexUpdCmd() {}
 
 void CortexUpdCmd::Exec(std::string v) {
-  // TODO(sang) stop server if it is running
+  {
+    auto config = file_manager_utils::GetCortexConfig();
+    httplib::Client cli(config.apiServerHost + ":" + config.apiServerPort);
+    auto res = cli.Get("/health/healthz");
+    if (res) {
+      CLI_LOG("Server is running. Stopping server before updating!");
+      commands::ServerStopCmd ssc(config.apiServerHost,
+                                  std::stoi(config.apiServerPort));
+      ssc.Exec();
+    }
+  }
+  if (CORTEX_VARIANT == file_manager_utils::kNightlyVariant) {
+    if (!GetNightly(v))
+      return;
+  } else {
+    if (!GetProAndBeta(v))
```

Comment:
What does "Pro" here stand for?

Reply:
'product` to align with `CORTEX_VARIANT`

Reply:
Renamed function to `GetStableAndBeta`, thanks @dan-homebrew